docs/deployment.rst

```diff
@@ -0,0 +1,93 @@
+E4S Facility Deployments

```

Comment:
I think there can be some miscommunication between E4S Facility Deployments that are advertised in the facility user docs vs what is provided called **Paratools Deployment**. I think in context of this documentation since we are talking about Paratools deployment we may want to rewrite what the title should be. I leave that up to you.

It would be good to atleast put link to https://docs.nersc.gov/applications/e4s/ in case they want to cross reference between e4s docs and nersc e4s docs. Note that we have put link in our docs to the e4s user docs.

Reply:
@shahzebsiddiqui  What is the relationship between Paratools deployments and NERSC support? I was going to add this but wanted to make sure it's accurate first. `This deployment is maintained by Paratools {LINK} and may not be officially supported by facility staff.`

Reply:
Yeah that sounds about right. Why don't you write all your thoughts in the documentation and then we can review and update as necessary.

Reply:
I pushed some edits that I think cover all the bases. We might make additional changes to the organization of the entries are more systems/deployments are added.